### ISSUE TITLE
Fix a11y contrast in API docs v1

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/forms-api#104]
# Problem

Documentation text fails the contrast required to meet WCAG (as found by Pete)

# Potential solution

Update CSS for higher contrast.
Consider using GCDS styles/tokens.

# Links

GitHub pages site: https://cds-snc.github.io/forms-api/
CSS files: https://github.com/cds-snc/forms-api/tree/docs/docs/assets/css

**Comment by thiessenp-cds** [comment]
The left outline still fails contrast. You could try making the text bigger or slight bolder?

**Comment by samsadasivan** [comment]
For instances of grey, we can use #64748B 

<img width="778" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdy9QQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ba3f02bd1e628b51238272a701057fe450585c0a/image.png" alt="image.png" />

**Comment by samsadasivan** [comment]
Up to you @anikbrazeau but if you want a softer "Black" on white, you can use our slate colour #1E293B

 <img width="1624" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeExQQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--536403aee6e57ba96ad793ad05991514ecac00cd/image.png" alt="image.png" />

**Comment by anikbrazeau** [comment]
**Note -- branding colour palette:** 

Primary (dark purple) = #4b408ff
Accent (purple)  = #9d8cf2ff
Accent (light purple) = #cec5f8ff

<img width="846" alt="Screenshot 2024-11-20 at 10 30 33 PM" src="https://github.com/user-attachments/assets/b5edb3ad-c15b-4f9d-97b0-a56f8312fe85">

**Comment by srtalbot** [comment]
@anikbrazeau and @samsadasivan to connect

**Comment by anikbrazeau** [comment]
After conversation with Carolyn and Sam:

Decision: Use #64748B in footer links

```
a, a:link, a:visited {
    color: #64748B !important;
    border-bottom: 1px dotted #64748B;
    text-decoration: none;
```

**Comment by anikbrazeau** [comment]
Tested by Sam and it passes https://cds-snc.github.io/forms-api/home/
